azkaban-common/src/main/java/azkaban/executor/ExecutorDao.java

```diff
@@ -16,89 +16,68 @@
 
 package azkaban.executor;
 
-import azkaban.database.AbstractJdbcLoader;
 import azkaban.db.DatabaseOperator;
-import azkaban.metrics.CommonMetrics;
-import azkaban.utils.Props;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 import javax.inject.Inject;
 import javax.inject.Singleton;
-import org.apache.commons.dbutils.QueryRunner;
 import org.apache.commons.dbutils.ResultSetHandler;
 import org.apache.log4j.Logger;
 
 @Singleton
-public class ExecutorDao extends AbstractJdbcLoader {
+public class ExecutorDao {
 
   private static final Logger logger = Logger.getLogger(ExecutorDao.class);
   private final DatabaseOperator dbOperator;
 
   @Inject
-  public ExecutorDao(final Props props, final CommonMetrics commonMetrics,
```

Comment:
This is great! I really like the simplification of dependencies. Also +1 for removing the `extends` as well.